src/modules/timepix3/configs/timepix3.json

```diff
@@ -2,5 +2,6 @@
   "Detector": "timepix3",
   "XResolution": 256,
   "YResolution": 256,
+  "NumberOfChunks": 1,
```

Comment:
number of partitions?  of the detector surface
add comment?
"comment" : "NumberofChunks - is the fjdkfdjkdjfdkjfd",